recipes-mender/mender/mender_0.1.bb

```diff
@@ -32,6 +32,7 @@ SYSTEMD_SERVICE_${PN} = "mender.service"
 FILES_${PN} += "${systemd_unitdir}/system/mender.service \
                 ${sysconfdir}/mender.conf \
                "
+SYSTEMD_AUTO_ENABLE = "disable"
```

Comment:
I think we should not make patches that knowingly break production functionality, even if it is temporary. Either the tests should build a separate image, or modify their own copy of the original (which I believe they do ATM).